api/filters/imagetag/updater.go

```diff
@@ -32,6 +32,9 @@ func (u imageTagUpdater) Filter(rn *yaml.RNode) (*yaml.RNode, error) {
 	if u.ImageTag.NewName != "" {
 		name = u.ImageTag.NewName
 	}
+	if u.ImageTag.TagSuffix != "" {
+		tag += u.ImageTag.TagSuffix
+	}
```

Comment:
Please add a test for the new feature in `api/filters/imagetag/imagetag_test.go`.

Reply:
added a test